.github/workflows/daily-test.yml

```diff
@@ -0,0 +1,55 @@
+name: Daily Test
+
+on:
+  schedule:
+    # More information on cron https://crontab.guru/
+    # GitHub actions is using UTC time. Scheduling action at 5 am PST
+    - cron: '0 13 * * *'
+
+  workflow_dispatch:
+
+env:
+  SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+  SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Remove unused environment variables

Reply:
It is used for `Send Notification` step

Reply:
these are global level env variables

Reply:
I see so these are used in third-party Github Actions libraries and also our awslabs/amazon-chime-sdk-apple package. Might be good to add a comment for Github Actions noobs like me, but of course that's nitpicking. Approved.